src/main.js

```diff
@@ -144,6 +145,20 @@ let load = loader({
     }),
   },
 
+  syncInstallations: {
+    requires: ['github', 'OwnersDirectory'],
+    setup: async ({github, OwnersDirectory}) => {
+      let gh = await github.getIntegrationGithub();
+      let installations = await gh.integrations.getInstallations({});
```

Comment:
We chose not to call this function on every `isRepoInstalled` call because it may someday contain 100's of installations.  If github eventually adds a `getInstallation(owner, repo)` method, we could use that.